hc/api/models.py

```diff
@@ -195,7 +205,10 @@ def notify(self, check):
 
         if error != "no-op":
             n = Notification(owner=check, channel=self)
-            n.check_status = check.status
+            if check.status.lower() == "up" and check.ping_is_early:
```

Comment:
`ping_is_early` is a method in the `Check` model. Right now, you have used it as an attribute.